build.sbt

```diff
@@ -13,15 +13,18 @@ lazy val suffuse = ( project in file(".")
 
 lazy val core = project dependsOn `fuse-jna` settings common
 
-def common = Seq[Setting[_]](
-                organization :=  "org.improving",
-    scalacOptions in compile ++= Seq("-Ywarn-unused", "-Ywarn-unused-import"),
-         libraryDependencies +=  "com.novocode" % "junit-interface" % "0.11" % "test",
-                    licenses :=  Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0")),
-                scalaVersion :=  "2.11.8",
-                 logBuffered :=  false,
-                   maxErrors :=  15,
-            triggeredMessage :=  Watched.clearWhenTriggered
+def common: Seq[Setting[_]] = Def settings (
+                           organization :=  "org.improving",
+                          scalacOptions ++= Seq("-feature", "-language:_", "-unchecked"),
+        inBoth(scalacOptions in compile ++= Seq("-Ywarn-unused", "-Ywarn-unused-import")),
```

Comment:
It's in both because we want those warnings when compiling both test and non-test sources.

`in compile`.. `in Compile` we can talk about that if you really want.. :-) But, no, it's right as it is here.